app/main.py

```
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_command = command.split()
+    if len(list_command) == 3 and list_command[0] == "cp":
+        cp, file, file_new = list_command
+        if file != file_new:
+            try:
+                with (
+                    open(file, "r") as file_in,
+                    open(file_new, "w") as file_out
+                ):
+                    content = file_in.read()
+                    file_out.write(content)
+            except FileNotFoundError:
+                print("file not found")
```

This violates checklist item #3: 'Remove any debug prints, comments, or helper/testing code before final submission.' The function should not print anything on error; it should silently do nothing if an error occurs.